synapse/lib/httpapi.py

```diff
@@ -1418,17 +1411,18 @@ async def _runHttpExt(self, meth, path):
 
         except Exception as e:
             rcode = True
-            enfo = s_common.err(e)
-            logger.exception(f'Extended HTTP API {iden} encountered fatal error: {enfo[1].get("mesg")}')
+            errname, errinfo = s_common.err(e)
+            extra = core.getLogExtra(httpapi=iden, errname=errname, **errinfo)
```

Comment:
Populating error information here would be duplicating of the default exc_info handling done by logger.exception().

Reply:
Agreed. I think i might have tweaked this before settling on a consistent way to handle exception info in the formatter :+1: